Sources/Mustache/Transform.swift

```diff
@@ -102,6 +102,78 @@ extension Array: ComparableSequence where Element: Comparable {
     }
 }
 
+extension Set: MustacheTransformable {
+    /// Transform Set.
```

Comment:
These `MustacheTransformable` conformances were missing, and were affecting the tests that I wrote.